bin/hubot

```diff
@@ -111,6 +111,11 @@ else
           console.error "Error parsing JSON data from hubot-scripts.json: #{err}"
           process.exit(1)
 
+        console.warn "Loading scripts from hubot-scripts.json is deprecated and " +
```

Comment:
This should probably be inside the above try-catch (`scripts` could be undefined at this point)